BankingManagement/account/account.go

```diff
@@ -0,0 +1,68 @@
+package account
+
+import (
+	"fmt"
+	history "hello/Assignment/github.com/BankingManagement/TransactionHistory"
+	"time"
+
+	uuid "github.com/google/uuid"
+)
+
+type accInterface interface {
+	Create(id string, name string) bool
+	Withdraw(amount int) bool
+	Deposit(amount int) bool
+	History()
+	Delete()
+}
+type Account struct {
+	Name, Id    string
+	money       int
+	Trandetails []history.Transaction
+}
+
+func (acc *Account) Create(id string, name string) bool {
+	acc.Name = name
+	acc.Id = id
+	acc.money = 0
+	return true
+}
+func (acc *Account) Withdraw(amount int) bool {
+
+	if amount > acc.money {
+		return false
```

Comment:
define err := errors.New("insufficient account balance")
and return this error to method caller